ASPNetCoreMastersTodoList.API/Filters/PerformanceResourceFilter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ASPNetCoreMastersTodoList.API.Filters
+{
+    public class PerformanceResourceFilter : IResourceFilter
+    {
+        private readonly Stopwatch _stopwatch = new Stopwatch();
```

Comment:
hi, current implementation is prone to race condition issue. we recommend storing the stopwatch in the httpContext.Items which you can see on the Tip section of session 7 ppt. Thanks

Reply:
Updated